Changelog.md

```diff
@@ -6,6 +6,7 @@ Features:
  * Type Checker: Disallow uninitialized storage pointers as experimental 0.5.0 feature.
  * Support and recommend using ``emit EventName();`` to call events explicitly.
  * Syntax Analyser: Do not warn about experimental features if they do not concern code generation.
+ * Syntax Analyser: Do not warn about `pragma experimental "v0.5.0"`.
```

Comment:
I would extend it: "since they do not affect code generation". I know it is stated above, but better to be explicit.